test/TestServerProjects/paging/Generated/Operations/PagingRestClient.cs

```diff
@@ -1048,11 +1048,7 @@ public async ValueTask<Response<ProductResult>> GetMultiplePagesLROAsync(string
                 switch (message.Response.Status)
                 {
                     case 202:
-                        {
-                            using var document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
-                            var value = ProductResult.DeserializeProductResult(document.RootElement);
-                            return Response.FromValue(value, message.Response);
-                        }
```

Comment:
What made the return type change here? It is still the same status code.

Reply:
This is now treated as LRO and not a paging method so we don't care about the initial response type.